src/api/firebase.js

```diff
@@ -63,7 +64,10 @@ export function useShoppingListData(listPath) {
 	const [data, setData] = useState(initialState);
 
 	useEffect(() => {
-		if (!listPath) return;
+		if (!listPath) {
+			setData(initialState);
```

Comment:
nit: A warning is coming up that useEffect has a missing dependency for 'initialState'.

Reply:
I'll make that change tonight to make sure it's clear by the time we merge!